app/main/views/frameworks.py

```diff
@@ -57,7 +57,7 @@ def update_framework(framework_slug):
                 data={'update': json_payload['frameworks']})
         )
         db.session.commit()
-    except (IntegrityError, ValidationError) as e:
+    except IntegrityError as e:
         db.session.rollback()
         abort(400, "Validation Error: {}".format(e))
```

Comment:
Good point, changed to `Database Error`, since that's what we use for other `IntegrityError` exceptions